src/views/PurchaseOrderDetail.vue

```diff
@@ -149,8 +149,10 @@
     <ion-footer>
       <ion-toolbar>
         <ion-item slot="end">
+        <ion-label>
```

Comment:
This is the not the correct approach to solve this case. For solving this issue in the correct way, you can once check the docs for ion-item.